app.rb

```diff
@@ -1,9 +1,29 @@
 require 'sinatra/base'
+require 'sinatra/reloader'
+require './lib/message'
 
 class Chitter < Sinatra::Base
+  configure :development do
+    register Sinatra::Reloader
+  end
+
   get '/test' do
     'Test page'
   end
```

Comment:
I understand why you would have a test page — but perhaps the next logical step would be to convert this to the homepage, with links to the 'get all messages', and 'create new message' routes.